storybook/stories/components/search/nav.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+import React from 'react'
+import { Col, Layout, Grid, Root } from 'reflex'
+import styles from '../../../shared/stories.css'
+
+export default function SearchNav() {
+  return (
+    <Layout className={styles.colors2} margin={[1, 0, 0, 0]}>
```

Comment:
hm could this be implemented with Grid padding instead of nesting the Grid? If the answer is no then I agree there is still a need for Layout margin